Libs/vtkSegmentationCore/vtkSegmentationModifier.cxx

```diff
@@ -198,6 +198,25 @@ void vtkSegmentationModifier::SeparateModifiedSegmentFromSharedLabelmap(vtkOrien
     }
 }
 
+inline int* GetMaximumExtent(const int *extent, vtkOrientedImageData* labelmap)
+{
+    static int maxExtent[6];
```

Comment:
Does the use of static variable significantly improve performances ?

Reply:
Not to my knowledge. It removes the following clang tidy warning : "Address of stack memory with local variable 'maxExtent' returned".

Reply:
it's not very threadsafe though... 
maybe pass output as an input parameter.

Reply:
I changed the method input args to take in the output extent instead.